### PR TITLE
Support markdown content

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi Team,

Sorry, I could not contain myself and I have added this PR, only for love.

The idea is accepting nice and safe markdown content in place of ugly and insecure HTML content.

The next step is finding a way to define the content (or part of it) without using this ugly JSON, maybe using independent files for entries or using yml.

Could you review it? 

@GusBaamonde, in the future we should do something with `<p>` styles, they do not make honor to parent element styles!

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "common": {
     "cancel": "Cancel",
-    "copyright_HTML": "© {year} Doppler LLC. All rights reserved. <a target=\"_blank\" href=\"https://www.fromdoppler.com/en/legal/privacy-policy\">Privacy and Legal Policy.</a>",
+    "copyright_MD": "© {year} Doppler LLC. All rights reserved. [Privacy and Legal Policy](https://www.fromdoppler.com/en/legal/privacy-policy).",
```
Comment (andresmoschini) [inline src/i18n/en.json]: ❤️

**Comment by cbernat** [comment]
> Hi Team,
> 
> Sorry, I could not contain myself and I have added this PR, only for love.
> 
> The idea is accepting nice and safe markdown content in place of ugly and insecure HTML content.
> 
> The next step is finding a way to define the content (or part of it) without using this ugly JSON, maybe using independent files for entries or using yml.
> 
> Could you review it?
> 
> @GusBaamonde, in the future we should do something with `<p>` styles, they do not make honor to parent element styles!

Maybe you could add a link to more information about this tool. What do you think? So it remains here for times to come.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -143,10 +144,11 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
         <footer>
           <CaptchaLegalMessage />
           <p>
-            <FormattedHTMLMessage
-              tagName="small"
-              id="common.copyright_HTML"
+            <FormattedMessageMarkdown
+              container="small"
```
Comment (cbernat) [inline src/components/Login/Login.js]: Why the change here? `tagname` by `container`

**Comment by cbernat** [comment]
👍

**Comment by andresmoschini** [comment]
> Maybe you could add a link to more information about this tool. What do you think? So it remains here for times to come.

There are a lot of markdown references:
* [The original one](https://daringfireball.net/projects/markdown/syntax)
* [The GitHub one](https://guides.github.com/features/mastering-markdown/)
* [The CommonMark one](https://commonmark.org/help/)

File: src/components/Login/Login.js (hunk 2 above)
Comment (andresmoschini) [reply src/components/Login/Login.js]: Because of `tagName` is an option of _FormattedMessage_ and _conatiner_ is a Parameter of _Markdown_ component (See [react-markdown options](https://github.com/rexxars/react-markdown#options)

**Comment by andresmoschini** [comment]
And there is also a demo of _react-markdown_ here: https://rexxars.github.io/react-markdown/
